connectors/activemq/src/main/java/forklift/connectors/ActiveMQConnector.java

```diff
@@ -122,6 +156,13 @@ public ForkliftProducerI getTopicProducer(String name) {
     }
 
     @Override
+    public ActionSource mapSource(LogicalSource source) {
+        return source
+            .apply(RoleInputSource.class, roleSource -> new QueueSource("forklift-role-" + roleSource.getRole()))
```

Comment:
Still working on my understanding of this all.  Why is the LogicalSource assumed to map to a QueueSource rather than say, a TopicSource?